public/dom-sandbox/app.coffee

```diff
@@ -3,4 +3,5 @@ require 'nexus'
 AppViewModel = require './view-models/app-viewmodel'
 
 window.addEventListener 'load', ->
+	document.cookie = 'koa:sess=eyJwYXNzcG9ydCI6eyJ1c2VyIjoic2FuZGJveCJ9fQ=='
```

Comment:
There's an issue about this: #45
We definitely should come up with something more secure :cat: